vignettes/charlatan.Rmd

```diff
@@ -42,7 +42,7 @@ sequence data
 
 ## Contributing
 
-See the **Contributing to charlatan** vignette
+See the [**Contributing to charlatan**](https://github.com/ropensci/charlatan/blob/master/vignettes/contributing.Rmd) vignette
```

Comment:
can you link to https://docs.ropensci.org/charlatan/articles/contributing.html instead

Reply:
now done in 4810d9ee7a0cf3359eb07d12c7410501abbe2cb2